android/app/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/view/FindMealContainerFragment.kt

```diff
@@ -0,0 +1,54 @@
+package com.kcc.kmmhackathon.androidHackathonApp.view
```

Comment:
I'm confused as to why you've created this file - please could you give some more context?

Reply:
Good point! But following that logic, the `FindMeal` also shouldn't know about creating new meals and creating a new container just for that seems a bit overkill.

I'd add this logic in the Activity holding the Fragments and BottomNavigation, maybe in a FloatingActionButton.

Either way, FAB or action in the Toolbar, the action belongs to the Activity, and then you have 3 independent Fragments to navigate to.